src/game/GameFSM.hpp

```diff
@@ -308,9 +308,7 @@ class GameFSM : public afsm::def::state_machine<GameFSM> {
 
         using internal_transitions = transition_table <
         // Event                                           Action
-        // Reply to MetaInformation request at any time during the game
-        in<spy::network::messages::RequestMetaInformation, actions::sendMetaInformation>,
-        in<spy::network::messages::Hello,         actions::HelloReply,    guards::isSpectator>>;
```

Comment:
warum wurde hier Hello->HelloReply gelöscht?

Reply:
Weil ich diese Transition mit in die internal_transitions von der Server FSM verschieben wollte, da Zuschauer ja immer joinen können. Allerdings habe ich an der Stelle vergessen sie auch wieder einzufügen. Danke für den Hinweis :+1: